Client/examples/client_lib_basic_example.cpp

```diff
@@ -0,0 +1,120 @@
+#include "ClientConfiguration.h"
+#include "cmd_arg_parse.h"
```

Comment:
nit picking, only `ClientConfiguration.h` and `chronolog_client.h` are required for a client app. `cmd_arg_parse.h` is for the command line argument parsing. `chrono_monitor.h` is for logging. both of them can be skipped. so, let's put the required ones together at the top. and put the optional ones after that.